CONTRIBUTING.md

```diff
@@ -161,10 +161,18 @@ Procedure of changing the image for the gatsby-image is the same as we change fo
 ## Adding Images and Icons
 
   1. It is recommended to use `@react-icons/all-files` instead of `react-icons` for importing icons. The issue with react-icons is that even though we are importing a single icon from the package/folder it still imports the complete icon folder which is creating unwanted junks of JS in the build.
-  2. It is recommended to use Gatsby's `<StaticImage>` instead of `<img>` tag to display static images on the site. This performs automatic image optimization, thereby improving site performance.
+  2. It is recommended to use Gatsby's `<StaticImage>` instead of `<img>` tag to display static images on the site except svg images. This performs automatic image optimization, thereby improving site performance.
```

Comment:
```suggestion
  2. It is recommended to use Gatsby's `<StaticImage>` instead of `<img>` tag to display static images on the site **except for SVG images**. This performs automatic image optimization, thereby improving site performance.
```